### PR TITLE
Evaluators MH

### PR DESCRIPTION
**Pull request by maciejhubert**

### CODE REVIEW

File: content/qa/skills/api_testing.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 +++
 title = "API testing"
+...
```
Comment (qbart) [inline content/qa/skills/api_testing.md:3]: please remove dots ✂️

File: content/qa/skills/ci_basics.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 +++
 title = "CI basics"
+...
```
Comment (qbart) [inline content/qa/skills/ci_basics.md:3]: please remove dots ✂️

File: content/qa/skills/test_automation.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 +++
 title = "Test automation"
+...
```
Comment (qbart) [inline content/qa/skills/test_automation.md:3]: please remove dots ✂️

**Comment by maciejhubert** [comment]
@qbart done

**Review by qbart — APPROVED** [review]

**Pull request merged**